packages/schematics/angular/workspace/files/tslint.json

```diff
@@ -1,32 +1,25 @@
 {
+  "extends": "tslint:recommended",
   "rulesDirectory": [
     "codelyzer"
   ],
   "rules": {
-    "arrow-return-shorthand": true,
-    "callable-types": true,
-    "class-name": true,
-    "comment-format": [
-      true,
-      "check-space"
-    ],
-    "curly": true,
+    "adjacent-overload-signatures": false,
```

Comment:
Looks like in the current CLI and tslint this has value `true`, lmk if I'm wrong.

Reply:
Hmm, I don't think it is present in the current tslint.json, so I guess it's not activated (unless I'm missing something?).